### PR TITLE
[IA-2328] Update images to reflect Bioconductor 3.12 release

### PR DESCRIPTION
**Pull request by nturaga**
https://broadworkbench.atlassian.net/browse/IA-2328

This PR updates the images for Bioconductor release 3.12.

- Should I be editing the changelog as well? They seem to be autogenerated.

### CODE REVIEW

**Comment by rtitle** [comment]
The change looks fine to me. 

Yes, can you update the `terra-jupyter-r` and `terra-jupyter-bioconductor` changelogs and also https://github.com/DataBiosphere/terra-docker/blob/master/config/conf.json? We do have a [script](https://github.com/DataBiosphere/terra-docker/blob/master/updateVersions.sc) to automate this but I think that's more for base image changes.

**Comment by nturaga** [comment]
> The change looks fine to me.
> 
> Yes, can you update the `terra-jupyter-r` and `terra-jupyter-bioconductor` changelogs and also https://github.com/DataBiosphere/terra-docker/blob/master/config/conf.json? We do have a [script](https://github.com/DataBiosphere/terra-docker/blob/master/updateVersions.sc) to automate this but I think that's more for base image changes.

Got it. I'll do that now and update the PR.

**Comment by nturaga** [comment]
@rtitle Let me know if this works.

**Review by Qi77Qi — APPROVED** [review]

**Review by rtitle — APPROVED** [review]
Looks good - I would say we can merge then let CI build the images, update Leo, and test.

**Comment by rtitle** [comment]
Also made https://broadworkbench.atlassian.net/browse/IA-2328 to track getting this tested & integrated into Leo

**Comment by rtitle** [comment]
@nturaga just checking, is this mergeable or are you still testing?

**Comment by nturaga** [comment]
> @nturaga just checking, is this mergeable or are you still testing?

Please merge! I've tested.

**Comment by rtitle** [comment]
Just realized `terra-jupyter-gatk` and `-aou` also depend on `terra-jupyter-r`, I'm going to tweak the changelogs then merge.

**Comment by rtitle** [comment]
@calbach just FYI about this change to the R image, which will cause `terra-jupyter-aou:1.0.16` to be built.

`terra-jupyter-aou:14` and `:11` will continue to be cached in GCE and Dataproc until you tell us otherwise.

**Comment by Qi77Qi** [comment]
if aou has upgraded to 14, we should be able to cache 1.0.16 and 1.0.14 now?

**Comment by rtitle** [comment]
Hm.. maybe that's a good idea @Qi77Qi. 

I can confirm and do that in the Leo PR once it's generated. Going to merge this for now. :)

File: config/conf.json [hunk 1]
```diff
@@ -104,7 +104,7 @@
             "packages" : {
                 
             },
-            "version" : "1.0.8",
+            "version" : "1.0.9",
```
Comment (Qi77Qi) [inline config/conf.json:107]: ah, thanks for updating the script..I think you need terra-docker-r under https://github.com/DataBiosphere/terra-docker/blob/a4bf66761880f95b0453a89c8bee682c92b29316/updateVersions.sc#L76 as well

**Pull request merged**